Config.php

```diff
@@ -53,6 +54,8 @@ class Config
         'ldap_alias_field' => array('aliasField'),
         'ldap_mail_field' => array('mailField'),
         'use_webserver_auth' => array('useKerberos'),
+        'required_member_of_field' => array('memberOfField'),
```

Comment:
This array is only for backwards compatibility w/ pre 3.X versions. New config options shouldn't have an entry here, please remove this line.